cmd/populator-controller/populator-controller.go

```diff
@@ -159,3 +165,24 @@ func getVolumePath(rawBlock bool) string {
 		return mountPath + "disk.img"
 	}
 }
+
+func getResources() (*corev1.ResourceRequirements, error) {
+	cpuLimit := settings.Settings.Migration.PopulatorContainerLimitsCpu
+	memoryLimit := settings.Settings.Migration.PopulatorContainerLimitsMemory
+	cpuRequest := settings.Settings.Migration.PopulatorContainerRequestsCpu
+	memoryRequest := settings.Settings.Migration.PopulatorContainerRequestsMemory
+	parsedCpuLimit, _ := resource.ParseQuantity(cpuLimit)
```

Comment:
You still don't need to do this `resource.ParseQuantity` as it's done inside the settigns